modules/apps/frontend-js/frontend-js-react-web/src/main/resources/META-INF/resources/js/index.es.js

```diff
@@ -12,18 +12,21 @@
  * details.
  */
 
-(function () {
-	const getModule = window[Symbol.for('__LIFERAY_WEBPACK_GET_MODULE__')];
+import process from 'process';
```

Comment:
I think we can drop the `.es.js` and just call this `.js`. It's about time. Same for all other files. 

We can do that later if we want to get this through, though.

Reply:
Yes, better doing it at the end. Every little change I do to this PR ends up eating one week of work :roll_eyes: